bootstrap/secret/insecure.go

```diff
@@ -26,6 +28,12 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/v3/clients/logger"
 )
 
+const (
+	// todo: should these be in `common` package?
+	writableKey        = "Writable"
```

Comment:
Make the one here public and use it. 
https://github.com/edgexfoundry/go-mod-bootstrap/blob/main/bootstrap/config/config.go#L52

Reply:
ok with the other InsecureSecrets specific constants being here.